spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -278,6 +278,10 @@ sealed class SpdxSingleLicenseExpression : SpdxExpression() {
         fun parse(expression: String): SpdxSingleLicenseExpression =
             SpdxExpression.parse(expression) as SpdxSingleLicenseExpression
     }
+    /**
+    * The URL that points to list of spdx licenses on spdx.org
+    */
+    val BASE_URL = "https://spdx.org/licenses/"
```

Comment:
I think I would prefer `SPDX_BASE_URL`.

Reply:
And how about moving it to the `SpdxConstants` object?

Reply:
Renamed and moved